test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -61,14 +65,17 @@ if (cluster.isMaster) {
     // an error is expected on the second worker
     process.send(`socket2:${err.code}`);
   });
-
+  
   socket1.bind({
     address: 'localhost',
-    port: common.PORT,
+    port: port,
     exclusive: false
   }, () => {
-    socket2.bind({ port: common.PORT + 1, exclusive: true }, () => {
-      // the first worker should succeed
+    port = socket1.address().port;
+    socket2.bind({
+      port: port + 1,
```

Comment:
Very true, I am trying to find some way to remove this type of syntax from the test.